core/src/main/java/com/redhat/lightblue/client/request/data/DataDeleteRequest.java

```diff
@@ -31,6 +31,7 @@ public JsonNode getBodyJson() {
         if (queryExpression != null) {
             node.set("query", queryExpression.toJson());
         }
+        appendExecutionToJson(node);
```

Comment:
It might be better if getBodyJson is implemented in abstractLightblueDataRequest as a method that creates a json node, and adds execution to it, and in all the subclasses, the overriding getBodyJson would first call super.getBodyJson(), and then add its bits to it.